### DISCUSSION TITLE
TreeView Selection Update

### DISCUSSION THREAD

**Discussion by enisn** [enisn/UraniumUI#510] [Announcements]
Happy to introduce that TreeView has built-in line selection now. There was already checkbox selection, but it's a different approach.

![image](https://github.com/enisn/UraniumUI/assets/23705418/91212871-2725-4319-b4f9-63b092ddc828)

![image](https://github.com/enisn/UraniumUI/assets/23705418/e247f3c0-0265-4ac9-80ff-7dfe6a934246)

You can watch the tutorial:
https://youtu.be/OUPiPGqotMc?si=787-8bRvHv56KGp5

You can check the documentation:
https://enisn-projects.io/docs/en/uranium/latest/themes/material/components/TreeView#selection

**Comment by mos379** [comment]
I'm not able to get a propertychanged notification when I use multiple selection and bind selectedNodes.
If the event would be public, then maybe I could use this to have access after selectionUpdate
